Tools/scripts/runcoptertest.py

```diff
@@ -35,7 +35,7 @@ def wait_time(mav, simtime):
         if t2 - t1 > simtime:
             break
 
-cmd = '../Tools/autotest/sim_vehicle.py -D'
+cmd = '../autotest/sim_vehicle.py -D'
```

Comment:
if you want to fix it, you shouldn't use relative path but absolute path from ArduPilot root directory.